WeakAuras/WeakAuras.lua

```diff
@@ -204,26 +204,17 @@ local globalDynamicConditionFuncs = {};
 -- All regions keyed on id, has properties: region, regionType, also see clones
 WeakAuras.regions = {};
 local regions = WeakAuras.regions;
-WeakAuras.auras = {};
-local auras = WeakAuras.auras;
-WeakAuras.events = {};
-local events = WeakAuras.events;
 
 -- keyed on id, contains bool indicating whether the aura is loaded
 WeakAuras.loaded = {};
 local loaded = WeakAuras.loaded;
 
-WeakAuras.specificBosses = {};
-local specificBosses = WeakAuras.specificBosses;
-WeakAuras.specificUnits = {};
-local specificUnits = WeakAuras.specificUnits;
-
 -- contains regions for clones
 WeakAuras.clones = {};
 local clones = WeakAuras.clones;
 
 -- Unused regions that are kept around for clones
-WeakAuras.clonePool = {};
+Private.clonePool = {};
```

Comment:
I've seen one aura in the past that this would be a breaking change for. But I don't think that's cause for concern.

Reply:
Oh. 

Without looking at the code, I'm pretty sure that's a point in favor of moving it to private.